### DISCUSSION TITLE
RSS feed

### DISCUSSION THREAD

**Discussion by PirroFrIta** [usememos/memos#4150] [Q&A]
Hi, is it possible to create an RSS feed automatically from a public memo feed?

**Comment by RoccoSmit** [comment]
There are 2 RSS feeds

Explore feed: `/explore/rss.xml`
User feed: `/u/{username}/rss.xml`

**Reply by jphastings** [reply]
I'd love for these feeds to be auto-discoverable — I've [added an issue for this](https://github.com/usememos/memos/issues/4276) 😊

**Comment by PirroFrIta** [comment]
Thank you. It’s exactly what I was looking for
